java/src/main/native/CMakeLists.txt

```diff
@@ -173,8 +173,8 @@ message(STATUS "RMM: RMM_INCLUDE set to ${RMM_INCLUDE}")
 
 find_path(ARROW_INCLUDE "arrow"
           HINTS "$ENV{ARROW_ROOT}/include"
-                "$ENV{CONDA_PREFIX}/include"
-                "${CUDF_CPP_BUILD_DIR}/_deps/arrow-src/cpp/src")
+                "${CUDF_CPP_BUILD_DIR}/_deps/arrow-src/cpp/src"
+                "$ENV{CONDA_PREFIX}/include")
```

Comment:
👍 

I wonder if we should remove `$ENV{CONDA_PREFIX}/include`, if we know that we should be picking up `arrow` from `cpp/build/_deps`.

Reply:
Yeah, I think that's probably a better approach.  I dropped the conda environment hints for Arrow.